flink-runtime-web/web-dashboard/web/partials/jobs/job.html

```diff
@@ -32,7 +32,7 @@
       - 
       {{ job['end-time'] | amDateFormat:'YYYY-MM-DD, H:mm:ss' }}</span></div>
   <div ng-if="job.duration &gt; -1" title="{{job.duration | humanizeDuration:false}}" class="navbar-info last first">{{job.duration | humanizeDuration:true}}</div>
-  <div ng-if="job.state=='RUNNING' || job.state=='CREATED' || job.state=='RESTARTING'" class="navbar-info last first"><span ng-click="cancelJob($event)" class="navbar-info-button btn btn-default">Cancel</span></div>
+  <div ng-if="job.type=='STREAMING' && (job.state=='RUNNING' || job.state=='CREATED' || job.state=='RESTARTING')" class="navbar-info last first"><span ng-click="cancelJob($event)" class="navbar-info-button btn btn-default">Cancel</span></div>
```

Comment:
This is weird. The stop button code isn't actually here. Are you sure this file is up-to-date? Can you run gulp again?